app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from random import randint
+
+
+def flip_coin():
+
+    flips_result = {0: 0, 1: 0, 2: 0, 3: 0, 4: 0,
+                    5: 0, 6: 0, 7: 0, 8: 0, 9: 0, 10: 0}
+
+    for i in range(10000):
+        heads = [randint(0, 1) for _ in range(10)].count(1)
+        flips_result[heads] += 0.01
+
+    flips_result_rounded = {key: int(flips_result[key])
+                            for key in flips_result}
+
+    return flips_result_rounded
```

Comment:
I ran your code a 100 times and never had the number of iterations been greater than 97 while we need 10000. It just doesn't work. And why would you add 0.01 and then round it? Just add 1, remove rounding and return flips_result.